### PR TITLE
Fix images path issue and toml syntax highlighting

### PR DESCRIPTION
**Pull request by jeradrutnam**

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -70,8 +66,6 @@ nav:
 # Extensions
 markdown_extensions:
   - markdown.extensions.admonition
-  - markdown.extensions.codehilite:
```
Comment (sherenewso2) [inline en/mkdocs.yml]: why are we removing this extension?

File: en/mkdocs.yml [hunk 2]
```diff
@@ -70,8 +66,6 @@ nav:
...
-      guess_lang: false
```
Comment (sherenewso2) [inline en/mkdocs.yml]: Why are we removing this extension?

File: en/mkdocs.yml [hunk 3]
```diff
@@ -89,15 +83,16 @@ markdown_extensions:
   - pymdownx.keys
   - pymdownx.mark
   - pymdownx.smartsymbols
-  - pymdownx.superfences
```
Comment (sherenewso2) [inline en/mkdocs.yml]: Why are we removing this extension?

File: en/docs/tutorials/configuring-a-service-provider-for-adaptive-authentication.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ Follow the steps below to set up the sample application to try out
 adaptive authentication using a sample application.
 
 1.  PickUp sample web application (SAML).
-    1.  Download the [saml2-web-app-pickup-dispatch.com.war](/assets/attachments/tutorials/saml2-web-app-pickup-dispatch.com.war)
+    1.  Download the [saml2-web-app-pickup-dispatch.com.war](../../assets/img/tutorials/saml2-web-app-pickup-dispatch.com.war)
```
Comment (sherenewso2) [inline en/docs/tutorials/configuring-a-service-provider-for-adaptive-authentication.md]: This is a .war file. It should be under 'attachments' folder.

Comment (jeradrutnam) [reply en/docs/tutorials/configuring-a-service-provider-for-adaptive-authentication.md]: Fixed

File: en/mkdocs.yml (hunk 2 above)
Comment (jeradrutnam) [reply en/mkdocs.yml]: This conflicts with highlightjs

File: en/mkdocs.yml (hunk 3 above)
Comment (jeradrutnam) [reply en/mkdocs.yml]: This conflicts with highlightjs, it intends to use pygments instead

**Pull request merged**